CHANGELOG.md

```diff
@@ -1,3 +1,66 @@
+# v0.5.0 (2020-08-13)
+
+Special thanks to first-time contributor @spoonofpower ([#988])!
+
+## Breaking changes
+
+* Remove support for unsigned datastore migrations ([#976])
+
+## OS changes
+
+* Add `aws-ecs-1` variant prototype for running containers in ECS clusters ([#946], [#1005], [#1007], [#1008], [#1009], [#1017])
+* Configurable `clusterDomain` kubelet setting via `settings.kubernetes.cluster-domain` ([#988])
+* Make update position within waves consistent ([#993])
+* Fix kubelet configuration for `MaxPods` ([#994])
+* Update `eni-max-pods` with new instance types ([#994])
+* Fix `max_versions` unit test in `updata` ([#998])
+* Remove injection of `label:disable` option for privileged containers in Docker ([#1013])
```

Comment:
Also up with the ecs item?

Reply:
This also applies to `aws-dev` variant since it also uses docker.